docs/README.rst

```diff
@@ -252,6 +254,28 @@ Some of these cert/key pairs can be wildcard (\*.mycf.com).
 
 .. _health checks:
 
+JSESSIONID Session Persistence
+------------------------------
+
+The |cfctlr| supports session persistence on the BIG-IP through cookies. Enabled by default on the |cfctlr|. To disable session persistence define the ``session_persistence`` setting as follows:
+
+.. code-block:: yaml
+
+   session_persistence: false
+
+To configure session persistence for the |cfctlr| your application must set cookies in its HTTP response headers as follows:
+
+``Set-Cookie: JSESSIONID=<value>; Max-Age=<age>``
+
+.. note::
+   - The cookie name of JSESSIONID is case insensitive so jsessionid and JSessionID are also valid.
+   - The cookie value can be anything as long as it is distinguishable from other cookies set by other responses.
```

Comment:
The cookie value can be any distinct value (in other words, be sure it's not the same as other cookies set by other responses).